.github/CODEOWNERS

```diff
@@ -22,10 +22,10 @@
 /pkg/version/                           @DataDog/croissant
 
 /Dockerfiles/                           @DataDog/kenafeh
-/pkg/collector/autodiscovery/           @DataDog/kenafeh @DataDog/croissant
+/pkg/autodiscovery/                     @DataDog/kenafeh @DataDog/croissant
+/pkg/autodiscovery/listeners            @DataDog/kenafeh
```

Comment:
based on the doc I think the trailing slash is important to define a directory: `/pkg/autodiscovery/listeners/`